src/main/java/ch/admin/bag/covidcertificate/sdk/core/extensions/DateTimeExtensions.kt

```diff
@@ -0,0 +1,15 @@
+package ch.admin.bag.covidcertificate.sdk.core.utils
+
+import java.time.Instant
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+
+val DEFAULT_DISPLAY_RULES_TIME_FORMATTER: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd")
+
+fun Instant.prettyPrint(dateFormatter: DateTimeFormatter): String {
+	return try {
+		this.atZone(ZoneId.systemDefault()).format(dateFormatter)
```

Comment:
should this not be formatted in UTC-Timezone?

Reply:
changed